collector/lib/CollectorConfig.h

```diff
@@ -196,6 +206,7 @@ class CollectorConfig {
   std::vector<double> connection_stats_quantiles_;
   double connection_stats_error_;
   unsigned int connection_stats_window_;
+  int64_t per_container_rate_limit_ = 1024;
```

Comment:
maybe make this a constant / env var

Reply:
I wonder how to determine a good default value here. At first, 1024 seemed a small value to me, however, after afterglow, it already takes a quite specific workload to reach to that value under 30s.

Reply:
I think this is Okay for now. We can adjust after doing performance tests.